cmd/gravy/simulate.go

```diff
@@ -76,13 +78,13 @@ func Simulate(stores map[string]db.Store, seed float64, output string) {
 	algorithm := algorithm.NewTradingAlgorithm(stores, exchange)
 
 	// Create output file.
-	out := gravyutil.FileWriterOrDie(output)
+	out := gravyutil.FileOrDie(output)
+	defer out.Close()
 
 	// Iterate over dates and simulate trading, export CSV.
 	WriteCSVHeader(algorithm.Headers(), out)
-	skipUntilIndex := 3650
 	hideAfterIndex := len(dates) / 2
-	for i := skipUntilIndex; i < len(dates); i++ {
+	for i := skip; i < len(dates); i++ {
```

Comment:
Self review: At least add a comment that we can save some time by not loading these records from disk (pass skip to GetDataStores).

Reply:
Done.

Reply:
Done.